moveit_ros/perception/handeye_calibration_solver/cmake/Modules/FindNumPy.cmake

```diff
@@ -0,0 +1,55 @@
+# - Find the NumPy libraries
```

Comment:
@v4hn does this FindNumPy approach make sense to you?

Reply:
I borrowed this `.cmake` file from the `caffe` project [link](https://github.com/BVLC/caffe/blob/master/cmake/Modules/FindNumPy.cmake), which is protected under BSD 2-clause license. I update this file with the copyright and license.